js/jquery.dbRecord.js

```diff
@@ -187,6 +187,8 @@
 		    data[name] = new Array(data[name], value);
 		}
 	    });
+            // Add the authenticity token.
+            data['_authenticity_token'] = $('[name=_authenticity_token]').val();
```

Comment:
If there is no hidden input field for authenticity_token what does `$('[name=_authenticity_token]').val()` evaluate to or does it raise an exception ?